docs-src/app-dev-context/parent-close-policy.md

```diff
@@ -8,3 +8,5 @@ tags:
 ---
 
 A [Parent Close Policy](/concepts/what-is-a-parent-close-policy) determines what happens to a Child Workflow Execution if its Parent changes to a Closed status (Completed, Failed, or Timed Out).
+
+The default Parent Close Policy option is set to `TERMINATE`.
```

Comment:
Because the option values differ across the SDKs, I recommend using prose—perhaps something like the following:

The default Parent Close Policy is to terminate the Child Workflow Execution.

Reply:
Updated to "The default Parent Close Policy option is set to terminate the Child Workflow Execution."